test/play_scraper_test.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+import numpy as np
```

Comment:
please add tests where the html attributes we are looking for are not present to make sure your code gracefully handles that